tests/debugging/mocking.py

```diff
@@ -33,18 +33,20 @@ def _write(self, payload):
         self.queue.append(payload.decode())
 
     def wait_for_payloads(self, cond=lambda _: bool(_), timeout=1.0):
```

Comment:
should we change the type of conde? to be lambda or number?

Reply:
currently there is no type